### PR TITLE
feat: specify TTL when deploying

### PR DESCRIPTION
**Pull request by tonsV2**

### CODE REVIEW

File: src/components/StackConfigurator.tsx [hunk 1]
```diff
@@ -69,6 +69,18 @@ export const StackConfigurator = forwardRef(function StackConfigurator(
...
+        ["1 hour", 60 * 60],
+        ["6 hours", 60 * 60 * 6],
+        ["12 hours", 60 * 60 * 12],
+        ["1 day", 60 * 60 * 24],
+        ["2 days", 60 * 60 * 24 * 2],
+        ["5 days", 60 * 60 * 24 * 5],
+        ["1 week", 60 * 60 * 24 * 7],
+        ["2 weeks", 60 * 60 * 24 * 7 * 2],
+        ["1 month", 60 * 60 * 24 * 7 * 4],
+    ]), [])
```
Comment (HendrikThePendric) [inline src/components/StackConfigurator.tsx:82]: This is purely a static constant so should be created outside of the function body. That way you also don't need `useMemo`. You would use `useMemo` if you want to create a memoized variable which is based on another variable which is only available within the function body, typically some prop or state variable.

**Review by HendrikThePendric — APPROVED** [review]
I'm approving because I know there's some time constraints to take into account, and because we're in different time-zones. However, please implement my suggestion before merging this in.

**Comment by HendrikThePendric** [comment]
Actually, my OCD got the better of me and I pushed a commit instead.

**Comment by tonsV2** [comment]
Thanks! Greatly appreciated :pray:

**Pull request merged**